openlibrary/accounts/model.py

```diff
@@ -681,13 +736,15 @@ def audit_accounts(email, password, test=False):
             ol_account.link(audit['link'])
             stats.increment('ol.account.xauth.auto-linked')
```

Comment:
Yes, this was an attempt to fix the problem where user logged in and it appeared as if they were not linked (when really the function to check link status had a regression). Resolved